win32_deps_build.sh

```diff
@@ -1,17 +1,10 @@
 #!/usr/bin/env bash
 
-set -e
+set -eu
 
 SCRIPT_DIR="$(dirname "$BASH_SOURCE")"
 SCRIPT_DIR="$(realpath "$SCRIPT_DIR")"
 
-USE_MINGW_LLVM=${USE_MINGW_LLVM:-}
```

Comment:
> and exported

I think it would nice to reduce the number of exports in `win32_build.sh` while at it.  For example, `OS` is passed to `win32_deps_build.sh` explicitly and probably doesn't need to be exported.  `USE_MINGW_LLVM` and `ENABLE_SHARED` could be passed explicitly too.  This would make the contract between `win32_build.sh` and `win32_deps_build.sh` clearer.

Reply:
Good point. I've dropped those exports, passing them explicitly. Also double checked to make sure that mingw-gcc still works.